packages/hardhat-core/src/internal/hardhat-network/stack-traces/solidity-stack-trace.ts

```diff
@@ -45,6 +46,7 @@ export interface SourceReference {
   contract?: string;
   function?: string;
   line: number;
+  range: [number, number];
```

Comment:
I don't get this. Why was this added? Pretty sure I'm missing something.

Reply:
This is used in `ErrorInferrer.filterRedundantFrames`